public/partials/signup.html

```diff
@@ -0,0 +1,101 @@
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label for="input3">Username</label>
+                            <input data-error="Please enter your username!" ng-model="user.us_username" type="text" class="form-control" id="username" placeholder="Username" required>
+                            <div class="help-block with-errors"></div>
+                        </div>
+                        <div class="form-group">
+                            <label for="input3">Email Address</label>
+                            <input data-error="Please enter a valid email address!" type="email" ng-model="user.us_email_address" type="text" class="form-control" id="input3" placeholder="Email Address" required>
+                            <div class="help-block with-errors"></div>
+                        </div>
+                        <div class="form-group">
+                            <label for="passwordField">Password</label>
+                            <input data-error="Please enter your chosen password!" ng-model="user.us_password" type="password" class="form-control" id="passwordField" placeholder="Password" required>
+                            <div class="help-block with-errors"></div>
+                        </div>
+                        <div class="form-group">
+                            <label for="input5">Confirm Password</label>
+                            <input data-match="#passwordField" data-error="Your passwords need to match!" ng-model="passwordConfirm" type="password" class="form-control" id="input5" placeholder="Confirm Password" required>
+                            <div class="help-block with-errors"></div>
+                        </div>
+                        <div class="form-group">
+                            <label for="input6">Phone Number</label>
+                            <input data-error="Please enter your phone number!" ng-model="user.us_contact" type="text" class="form-control" id="input6" placeholder="Phone Number" required>
+                            <div class="help-block with-errors"></div>
+                        </div>
+                        <div class="form-group">
+                            <label for="input7">Address</label>
+                            <input data-error="Please enter your address!" ng-model="user.us_address" type="text" class="form-control" id="input7" placeholder="Address">
+                            <div class="help-block with-errors"></div>
+                        </div>
+                        <div class="form-group">
+                            <label for="input8">Role Requested</label>
+                            <select ng-init="selectedRole=roles[0]" data-error="Please select a role!" class="form-control" ng-model="selectedRole" ng-change="user.us_user_role = selectedRole" ng-options="r._id as r.ro_role_name for r in roles" required>
```

Comment:
Hey @nick22891 I believe the flow was for the user to be first assigned a default role - not to request it at the beginning.  Remove this field from the registration screen.  I will provide a path for all the application defaults.  This was missing from my endpoints.  

I think in the application defaults there will be information you can use such as:
`{app_title: "Harvest v2", app_author: "SlashRoots Foundation", default_role: "57d438497a48b2cb165fc9f0",...}`